Add GPTQ quantization to executorch (not tested)

jerryzh168 opened the pull request:
Summary: will test in OSS and fix issues in a separate PR

Differential Revision: D54569842

digantdesai commented:
> (not tested)
LOL

digantdesai reviewed (approved):
Stamping, make sure CI is green (if relevant)

jerryzh168 committed:
Add GPTQ quantization to executorch (not tested) (#2274)

Summary:

will test in OSS and fix issues in a separate PR

bypass-github-export-checks
bypass-github-pytorch-ci-checks
bypass-github-executorch-ci-checks

Reviewed By: digantdesai

Differential Revision: D54569842
